### PR TITLE
make sparsity configurable

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- add `weight_sparsity: 0.8` to defaults
- part of https://github.com/RasaHQ/research/issues/62

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by Ghostvv** [comment]
@tabergma do you have an idea where an error is coming from? I didn't change anything in `tests`

**Comment by tabergma** [comment]
Already fixed in `tf2`, just merge the branch into yours. pycharm always creates imports without `rasa.` for me. Sorry about that.

File: rasa/nlu/selectors/response_selector.py [hunk 1]
```diff
@@ -52,7 +53,7 @@
 logger = logging.getLogger(__name__)
 
 
-class ResponseSelector(DIETClassifier):
+class ResponseSelector(DIETSelector):
```
Comment (tabergma) [inline rasa/nlu/selectors/response_selector.py]: `DIETSelector`?

Comment (Ghostvv) [reply rasa/nlu/selectors/response_selector.py]: this is PR before removing it

Comment (tabergma) [reply rasa/nlu/selectors/response_selector.py]: ok, lost the overview 🙈

**Comment by Ghostvv** [comment]
I'll merge these two PRs together, because resolving the same conflicts is tedious

**Pull request merged**